dspace-api/src/main/java/org/dspace/app/util/DCInputSet.java

```diff
@@ -107,9 +108,20 @@ public boolean isFieldPresent(String fieldName) {
         for (int i = 0; i < inputs.length; i++) {
             for (int j = 0; j < inputs[i].length; j++) {
                 DCInput field = inputs[i][j];
-                String fullName = field.getFieldName();
-                if (fullName.equals(fieldName)) {
-                    return true;
+                if (field.getInputType().equals("qualdrop_value")) {
```

Comment:
This entire `if` took me a moment to understand. I think we should add a comment here to describe why "qualdrop_value" has to be treated separately.  It might say something like:
`// If this is a "qualdrop_value" field, then the full field name is the field + dropdown qualifier`